solenoid_app/views.py

```diff
@@ -85,6 +85,7 @@ def formHandle(request):
 
 
 def voltageChart(request):
+    compute = ''
```

Comment:
Given that in each condition, compute eventually gets assignment, I'm not worried about this.

Reply:
However, it is strange that we never had the failure when manually testing.
Basically, when we'd run tests on `python_tests.test_urls`, we'd get a failure for `local variable 'compute' referenced before assignment`.